packages/blockchain/src/airdrop/claimSablierAirdrop.ts

```diff
@@ -0,0 +1,105 @@
+import { getPublicClient } from '@packages/blockchain/getPublicClient';
+import { getWalletClient } from '@packages/blockchain/getWalletClient';
+import { type Address, type Hash } from 'viem';
+import { privateKeyToAccount } from 'viem/accounts';
+
+import { checkSablierAirdropEligibility } from './checkSablierAirdropEligibility';
+
+export async function claimSablierAirdrop({
+  chainId,
+  contractAddress,
+  recipientAddress,
+  cid,
+  adminPrivateKey
+}: {
+  chainId: number;
+  contractAddress: Address;
+  recipientAddress: Address;
+  cid: string;
+  adminPrivateKey: `0x${string}`;
```

Comment:
We shouldn't need an admin key.

I'm assuming anyone can pay for gas to make a claim for anyone.

So the test is to use a wallet that has a bit of ETH but is not admin or mentioned in the airdrop